Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Compile
+FROM    alpine:3.10 AS compiler
+
+RUN     apk update --quiet
+RUN     apk add curl
+RUN     apk add build-base
+RUN     apk add libressl-dev
+
+RUN     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+WORKDIR /meilisearch
+
+COPY    . .
+
+ENV     RUSTFLAGS="-C target-feature=-crt-static"
```

Comment:
Why did you specified that line? To make it act like a fully static binary? Does it work?

Reply:
Ok, that make it compile in fact